antenna-documentation/scripts/documentation_release.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash -e
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+CURRENT_BRANCH=${git describe --tag}
```

Comment:
If you really only want the name of the current branch (if any), better use `git rev-parse --abbrev-ref HEAD`.

Reply:
I want the name of the tag to properly name folder names. 
The variable has the wrong name. I renamed it.